examples/private_scan.py

```diff
@@ -0,0 +1,89 @@
+"""
+Tool for scanning files privately using VirusTotal API.
+Supports waiting for scan completion.
+"""
+
+import sys
+import asyncio
+import argparse
+from pathlib import Path
+import vt
+from rich.console import Console
+from rich.progress import Progress
+
+console = Console()
+
+async def scan_file_private(
+    api_key: str,
+    file_path: Path,
+    wait: bool = False
+) -> None:
+    """
+    Scan a file privately on VirusTotal.
+    
+    Args:
+        api_key: VirusTotal API key
+        file_path: Path to file to scan
+        wait: Wait for scan completion
+    """
+    async with vt.Client(api_key) as client:
+        try:
+            with Progress() as progress:
+                task = progress.add_task(
+                    "Scanning file...",
+                    total=None if wait else 1
+                )
+
+                analysis = await client.scan_file_private_async(
+                    str(file_path),
+                    wait_for_completion=wait
+                )
+
+                progress.update(task, advance=1)
+
+                console.print("\n[green]Scan submitted successfully[/green]")
+                console.print(f"Analysis ID: {analysis.id}")
+
+                if wait:
+                    console.print(f"\nScan Status: {analysis.status}")
+                    if hasattr(analysis, 'stats'):
+                        console.print("Detection Stats:")
+                        for k, v in analysis.stats.items():
+                            console.print(f"  {k}: {v}")
+
+        except vt.error.APIError as e:
+            console.print(f"[red]API Error: {e}[/red]")
+        except Exception as e:
+            console.print(f"[red]Error: {e}[/red]")
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Scan file privately using VirusTotal API"
+    )
+    parser.add_argument("apikey", help="VirusTotal API key")
+    parser.add_argument("file_path", help="Path to file to scan")
```

Comment:
One last suggestion: I've tested this myself and I needed to add this change to my local copy in order to use the script like this:
```
$ private_scan.py --apikey="sdfsfs" --file_path="dfdgdf"
```

```suggestion
    parser.add_argument("--apikey", help="VirusTotal API key")
    parser.add_argument("--file_path", help="Path to file to scan")
```